### PR TITLE
8.0 hr public holidays

### PR DESCRIPTION
**Pull request by saltonmassally**

### CODE REVIEW

File: hr_public_holidays/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/hr/issues/new?body=module:%20hr_public_holidays%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Salton Massally (iDT Labs) <smassally@idtlabs.sl>
```
Comment (dreispt) [inline hr_public_holidays/README.rst:50]: What about the author?

File: hr_public_holidays/models/hr_public_holidays.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    ]
+
+    @api.one
+    @api.constrains('year')
+    def _check_year(self):
+        if self.search_count([('year', '=', self.year),
+                              ('country_id', '=', False),
+                              ('id', '!=', self.id)]):
+            raise ValidationError('You can\'t create duplicate year')
+        return True
```
Comment (dreispt) [inline hr_public_holidays/models/hr_public_holidays.py:62]: Isn't this check redundant with the SQL constraint?

File: hr_public_holidays/models/hr_public_holidays_line.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class HrHolidaysLine(models.Model):
+    _name = 'hr.holidays.public.line'
+    _description = 'Public Holidays Lines'
+    _order = "date, name desc"
+
+    name = fields.Char(
+        'Name',
+        size=128,
```
Comment (dreispt) [inline hr_public_holidays/models/hr_public_holidays_line.py:35]: remove size

File: hr_public_holidays/README.rst (hunk 1 above)
Comment (saltonmassally) [inline hr_public_holidays/README.rst:50]: Copied from one of my addons, forgot that section... adding it

File: hr_public_holidays/views/hr_public_holidays_view.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                        <group>
+                            <field name="year" />
+                        </group>
+                        <group>
+                            <field name="country_id" />
+                        </group>
+                    </group>
+                    <newline />
+                    <group string="Public Holidays" colspan="4"
+                        col="1">
```
Comment (dreispt) [inline hr_public_holidays/views/hr_public_holidays_view.xml:29]: remove colspan and col

File: hr_public_holidays/models/hr_public_holidays.py (hunk 2 above)
Comment (saltonmassally) [inline hr_public_holidays/models/hr_public_holidays.py:62]: It is need as the constraint is not triggered in cases that no country is supplied... will allow duplicate years as long as you do not supply country

**Comment by saltonmassally** [comment]
@dreispt  I just noticed that I created a pull request on 9.0 my intention was 8.0 who do I go about editing this? Thanks

File: hr_public_holidays/models/hr_public_holidays.py (hunk 2 above)
Comment (dreispt) [inline hr_public_holidays/models/hr_public_holidays.py:62]: strange, but ok then

**Comment by dreispt** [comment]
AFAIK you have to close this one and open a new PR

**Comment by saltonmassally** [comment]
thanks @dreispt ; moved to https://github.com/OCA/hr/pull/173

**Pull request closed without merging**